src/KnpUOAuth2ClientBundle.php

```diff
@@ -11,16 +11,15 @@
 namespace KnpU\OAuth2ClientBundle;
 
 use KnpU\OAuth2ClientBundle\DependencyInjection\KnpUOAuth2ClientExtension;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
 class KnpUOAuth2ClientBundle extends Bundle
 {
     /**
      * Overridden to allow for the custom extension alias.
-     *
-     * @return KnpUOAuth2ClientExtension
      */
-    public function getContainerExtension()
+    public function getContainerExtension(): ?ExtensionInterface
```

Comment:
`ExtensionInterface` is a different from that we had in `@return KnpUOAuth2ClientExtension` - we had incorrect return type? Or if that `KnpUOAuth2ClientExtension` still relevant, I'd recommend keeping that PHPDoc like this:
`@return KnpUOAuth2ClientExtension|ExtensionInterface`

Reply:
It's minor - but I agree :). But I'm going to merge this